blocks/card/card.css

```diff
@@ -114,7 +114,7 @@
 
 /* on smaller screens, show everything in a single column. */
 @media (max-width: 900px) {
-  .section.card-container {
+  .card-container.three-columns{
```

Comment:
Now this is only applied to 3-col layouts, but not to 2-col layout any more. See https://56-create-collections-pages--24life--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/block-library/blocks/card&index=1

You could comma-separate multiple selectors.

Reply:
oops I will fix this right away